### DISCUSSION TITLE
能否考虑加入快捷键

### DISCUSSION THREAD

**Discussion by FengLiuI** [Richasy/Bili.Uwp#1070] [想法]
就是在播放界面加入各种快捷键，就像bilibili网页端的一样。比如F：全屏/退出全屏，[：上一集，]：下一集，等等。
![QQ截图20220418103533](https://user-images.githubusercontent.com/51824395/163745595-1d6a1f99-a0b5-4d30-a392-64ba2ef45cf5.png)

**Comment by Richasy** [comment]
https://github.com/Richasy/Bili.Uwp/wiki/%E5%BF%AB%E6%8D%B7%E9%94%AE

**Reply by FengLiuI** [reply]
原来有啊，我以为没有呢

**Reply by Hodpel** [reply]
但是是否能考虑增大对于快捷键设定的自由度
一是支持快捷键自定义 如我本地视频多是用potplayer看视频,网页端也安装了类似操作的(油猴)脚本 (用惯了先前的快捷键直接转平台的话会比较难受(比如习惯上使用enter全屏结果是播放和暂停....)
二是丰富播放器快捷键量 下面给出一些可能的快捷键表(包括部分已有快捷键)(不过这些都是偏个人喜好了...不喜莫怪)
|  快捷键   | 说明    |
| :---: | :---: |
| <kbd>→</kbd> | 快进5秒 |
| <kbd>←</kbd> | 后退5秒 |
| <kbd>Ctrl</kbd>+<kbd>→</kbd> | 快进30秒 |
| <kbd>Ctrl</kbd>+<kbd>←</kbd> | 后退30秒 |
| <kbd>↑</kbd> | 音量升高 10% |
| <kbd>↓</kbd> | 音量降低 10% |
| <kbd>Ctrl</kbd>+<kbd>↑</kbd> | 音量升高 20% |
| <kbd>Ctrl</kbd>+<kbd>↓</kbd> | 音量降低 20% |
| <kbd>C</kbd> | 加速播放 +0.1 |
| <kbd>X</kbd> | 减速播放 -0.1 |
| <kbd>Z</kbd> | 恢复1倍速或上次播放速度 |
| <kbd>Enter</kbd> | 进入/取消全屏 |
| <kbd>D</kbd> | 上一帧 |
| <kbd>F</kbd> | 下一帧 |
| 数字1-6(包括数字区和小键盘区) | 快速调整1-6倍速 |
